src/main/resources/org/csanchez/jenkins/plugins/kubernetes/ContainerTemplate/config.jelly

```diff
@@ -23,11 +23,11 @@
   </f:entry>
 
   <f:entry field="command" title="${%Command to run}">
-    <f:textbox default="/bin/sh -c"/>
+    <f:textbox default="${descriptor.defaultCommand}"/>
   </f:entry>
 
   <f:entry field="args" title="${%Arguments to pass to the command}">
-    <f:textbox default="cat"/>
+    <f:textbox default="${descriptor.defaultArgs}"/>
   </f:entry>
 
   <f:entry field="ttyEnabled" title="${%Allocate pseudo-TTY}">
```

Comment:
Change the default to be false in this case. The only reason for `tty: true` was to pair with `cat`. IOW

```bash
cat
```

with no input, but with a TTY (so no EOF), is just an idiom for sleeping.

Reply:
done